### ISSUE TITLE
Error while deploying prometheus and labeling pods

### ISSUE THREAD

**Issue by JooyoungPark73** [vhive-serverless/invitro#333]
### Prometheus deployment
https://github.com/vhive-serverless/invitro/blob/66f1567d8efeae1144d03718ed9358e4807d3da4/config/kubeadm_init.yaml#L19
does not match current K8S version
should be: `kubernetesVersion: v1.25.9`

### Labeling
https://github.com/vhive-serverless/invitro/blob/66f1567d8efeae1144d03718ed9358e4807d3da4/scripts/setup/label.sh#L45
does not work on mac
should be: `sed -i'' '1d' tmp`

**Comment by leokondrashov** [comment]
I've checked the prometheus with new invitro version, it should be working. But yes, this should be updated.

I have no idea why mac is that weird about the sed stuff. By the description of sed, the suffix is optional. sed is half a century old and having different behaviour is ridiculous.

**Comment by JooyoungPark73** [comment]
for mac, sed -i requires suffix as mandatory. Well, This sed error is not critical, so I guess it is okay

**Comment by leokondrashov** [comment]
@JooyoungPark73 Is the sed part working? Because the kubeadm part is resolved by #351. Should we close it?

**Comment by JooyoungPark73** [comment]
It does not work on mac, but it spits out that there is no node named NODE, and proceed to node-0, so its okay to close it.
kubectl get nodes output:
```
NAME                                         STATUS   ROLES           AGE     VERSION
node-0.w-io.ntu-cloud-pg0.utah.cloudlab.us   Ready    control-plane   7d20h   v1.25.9
node-1.w-io.ntu-cloud-pg0.utah.cloudlab.us   Ready    <none>          7d20h   v1.25.9
node-2.w-io.ntu-cloud-pg0.utah.cloudlab.us   Ready    <none>          7d20h   v1.25.9
node-3.w-io.ntu-cloud-pg0.utah.cloudlab.us   Ready    <none>          7d20h   v1.25.9
node-4.w-io.ntu-cloud-pg0.utah.cloudlab.us   Ready    <none>          7d20h   v1.25.9
node-5.w-io.ntu-cloud-pg0.utah.cloudlab.us   Ready    <none>          7d20h   v1.25.9
```
